openedx/core/djangoapps/coursegraph/management/commands/dump_to_neo4j.py

```diff
@@ -90,11 +83,14 @@ def serialize_item(item):
 
         label = item.scope_ids.block_type
```

Comment:
The reason I say so is if it would have named as `item_block_type`, I would not have asked the following question.
```
What can be the other possibilities for a label here? Just curious.
```
Clearer right? :-)